src/FunderMaps.Infrastructure/Storage/SpacesBlobStorageService.cs

```diff
@@ -189,26 +194,109 @@ public async Task StoreDirectoryAsync(string directoryName, string directoryPath
         {
             try
             {
-                async Task DirectorySearch(string path, string subdir = "")
+                TransferUtilityUploadDirectoryRequest request = new()
+                {
+                    BucketName = _options.BlobStorageName,
+                    Directory = directoryPath,
+                    KeyPrefix = directoryName,
+                    SearchOption = SearchOption.AllDirectories,
+                    UploadFilesConcurrently = true,
+                };
+
+                request.UploadDirectoryFileRequestEvent += (sender, e) =>
```

Comment:
What is `e`? Write the full var name

Reply:
`e` is the full var name. 
![image](https://user-images.githubusercontent.com/8037254/103086543-2bd2e180-45e5-11eb-82e5-7ad2a2375ea9.png)